### PR TITLE
Fix website links and images

### PR DESCRIPTION
**Pull request by acdvs**
| Q                        | A  |
| ------------------------ | ---  |
| Fixed Issues?            | n/a |
| Patch: Bug Fix?          | No |
| Major: Breaking Change?  | No |
| Minor: New Feature?      | No |
| Tests Added + Pass?      | n/a |
| Any Dependency Changes?  | No |

- Several links and images were broken due to absolute paths. Changed to relative paths.
- Code blocks (especially on the home page) were being rendered without the correct style unless the page was refreshed.

### CODE REVIEW

**Comment by Andarist** [comment]
Sponsors are still not rendered correctly - they arrive in the SSRed HTML but are removed during React hydration (at least it looks like it). 🤔

File: website/src/components/CodeSnippet.jsx [hunk 1]
```diff
@@ -1,20 +1,27 @@
...
 
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import useThemeContext from '@theme/hooks/useThemeContext';
 
 function CodeSnippet({ code }) {
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
+  }, []);
```
Comment (Andarist) [inline website/src/components/CodeSnippet.jsx:15]: could you explain the rationale behind this? I understand what it does (remounts the `<Highlight/>`) but I'm wondering why this is needed here.

**Comment by acdvs** [comment]
Still looking into this. Everything shows up fine in development (via both `start` and `build` methods) but it's not translating to the production build for some reason.

File: website/src/components/CodeSnippet.jsx (hunk 1 above)
Comment (acdvs) [reply website/src/components/CodeSnippet.jsx:15]: It just ensures that the component and its contents are mounted _after_ the code style is loaded.

Comment (Andarist) [reply website/src/components/CodeSnippet.jsx:15]: what the code style (css for that?) is and how it relates to mounting?

**Comment by Andarist** [comment]
> Still looking into this. Everything shows up fine in development (via both start and build methods) but it's not translating to the production build for some reason. I would've marked this as a draft if I knew this beforehand.

Not sure what is the exact Babel-related setup used here regarding dev & prod environments (didn't really looked into it) but I've debugged this and:
```js
[...new Array(COUNT)]
```
was being transpiled to:
```js
// most likely caused by Babel's "loose" mode
[].concat(new Array(COUNT))
```
which broke the iteration as this has been producing an array with empty slots instead of an array with undefineds. I've fixed this by using `Array.from`. This probably breaks compatibility with older browsers but the site has been built using CSS vars etc so that ship has already sailed 😉

**Comment by acdvs** [comment]
Ah that makes sense. Didn't even consider how it was being transpiled. Thanks for the help!

**Pull request merged**